src/main/resources/interface-specification.yml

```diff
@@ -4988,6 +4988,8 @@ components:
           type: boolean
         suspendedByPlatform:
           type: boolean
+        contractDownload:
```

Comment:
I don't know the exact requirement, but why is this optional? I mean, whether the purpose of this field is to check whether or not the contract is present, this can be achieved by simply using the boolean value.

Reply:
+1

Reply:
+1

Plus, the name is a bit misleading as it is not intended to carry contract download information, I would suggest `isContractPresent`, or similar